bc_obps/registration/schema/v2/operation.py

```diff
@@ -39,6 +43,14 @@ class Config:
         ]
         from_attributes = True
 
+    @staticmethod
+    def resolve_sfo_facility_id(obj: Operation) -> Optional[UUID]:
+        # Resolve a single facility id for SFO operations
+        facilities = FacilityDataAccessService.get_current_facilities_by_operation(obj)
+        if obj.type == OperationTypes.SFO.value and len(facilities) > 0:
+            return facilities[0].id
+        return None
```

Comment:
This should be safer once we add some safeguards to ensure an SFO can't create multiple Facilities:

https://github.com/bcgov/cas-registration/issues/2105